src/applications/mhv/landing-page/components/HeaderLayoutV2.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
A suggestion here instead of having a v2 is to use the selector in the HeaderLayout V1. That way can we can later just remove the conditional there when the feature is permanent.